src/components/ProfileList/ProfileList.tsx

```diff
@@ -1,79 +1,52 @@
 'use client'
-import React, { useState, useEffect } from 'react'
+import React from 'react'
 import styles from '@/components/ProfileList/ProfileList.module.scss'
 import ProfilePicture from '@/assets/images/ProfilePicture.png'
 import {
   ProfileData,
   ProfileListItems,
 } from '@/components/ProfileList/ProfileData'
-
-enum JobSpecialization {
-  Frontend = 'Frontend',
-  Backend = 'Backend',
-  Fullstack = 'Fullstack',
-  None = 'None',
-}
-
-interface TechnologyChipProps {
-  name: string
-  theme: JobSpecialization
-  seniority: string
-}
-
-const getTechnologyClassByTheme = (
-  theme: JobSpecialization,
-  seniority: string,
-) => {
-  if (seniority.includes('Frontend')) {
-    return styles.frontend
-  } else if (seniority.includes('Backend')) {
-    return styles.backend
-  } else if (seniority.includes('Fullstack')) {
-    return styles.fullstack
-  }
-  switch (theme) {
-    case JobSpecialization.Frontend:
-      return styles.frontend
-    case JobSpecialization.Backend:
-      return styles.backend
-    case JobSpecialization.Fullstack:
-      return styles.fullstack
-    default:
-      return ''
-  }
-}
-const getTechnologyClassBySeniority = (seniority: string) => {
-  if (seniority.includes('Frontend')) {
-    return styles.frontend
-  } else if (seniority.includes('Backend')) {
-    return styles.backend
-  } else if (seniority.includes('Fullstack')) {
-    return styles.fullstack
-  }
-  return ''
-}
-
-const TechnologyChip: React.FC<TechnologyChipProps> = (props) => {
-  const themeClass = getTechnologyClassByTheme(props.theme, props.seniority)
-  return <p className={`${styles.badge} ${themeClass}`}>{props.name}</p>
-}
+import { useFilters } from '@/contexts/FilterContext'
+import classNames from 'classnames/bind'
+const cx = classNames.bind(styles)
 
 const ProfileListItem: React.FC<{ data: ProfileListItems }> = ({ data }) => {
-  const [theme, setTheme] = useState<JobSpecialization>(JobSpecialization.None)
+  const getStackClasses = cx({
+    [styles.frontend]: data.stack === 'Frontend',
```

Comment:
**nitpick:** You already duplicated these values few times, and you have enum for that: `JobSpecialization.Frontend`, let's use it everywhere